### PR TITLE
school-main

### PR DESCRIPTION
**Pull request by dalshekerchi**

### CODE REVIEW

**Review by EvgeniaSila — DISMISSED** [review]
Looks good

File: Uni_life_tracker/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -28,16 +28,30 @@
...
     <string name="lowercase">(?=.*[a-z])</string>
     <string name="number"
             tools:ignore="TypographyDashes">(?=.*[0-9])</string>
+
+    <!-- Calendar    -->
     <string name="specialChar"><![CDATA[(?=.*[@#!$%^&+=])]]></string>
     <string name="minLength">.{6,}</string>
+    <string name="sign_up_title">Sign Up</string>
+    <string name="welcome_to">Welcome to</string>
+    <string name="uni_life_tracker">Uni Life Tracker</string>
```
Comment (hellokathylee) [inline Uni_life_tracker/app/src/main/res/values/strings.xml]: remove uni life tracker

File: Uni_life_tracker/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -28,16 +28,30 @@
...
+    <string name="l_arrow"><![CDATA[<-]]></string>
```
Comment (hellokathylee) [inline Uni_life_tracker/app/src/main/res/values/strings.xml]: remove arrows & use icons instead?

**Review by hellokathylee — DISMISSED** [review]
look over & change ids to the naming conventions, otherwise its good!!

**Review by EvgeniaSila — APPROVED** [review]

**Review by hellokathylee — APPROVED** [review]

**Pull request merged**